roles/apache/tasks/ssl_redhat.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set stapling files
+  copy:
+    src: stapling.conf
+    dest: /etc/httpd/conf.d/stapling.conf
+    mode: 0644
+    owner: root
+    group: root
+  notify: restart apache
```

Comment:
feels like this could be merged into existing with some vars, but i'm not going to get picky with all this.

Reply:
I like that approach too. This is lazy, but it does work.